src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor

```diff
@@ -1,14 +1,14 @@
 ﻿
 <section class="page-section getting-started-step-section">
 
-    <div class="step-body-card">
+    <div id="getting-started" class="step-body-card">
```

Comment:
assigning `id` to the `div` will result into the `anchor`?

Reply:
yes, the spec:
https://www.w3.org/TR/html4/struct/links.html#h-12.2.3

Reply:
Yes, do you think any change need?